src/zino/tasks/linkstatetask.py

```diff
@@ -179,8 +179,8 @@ def _update_ifalias(self, port: Port, data: BaseInterfaceRow):
                 _logger.info(
                     "%s: changing desc for %s from %r to %r", self.device.name, data.index, port.ifalias, data.alias
                 )
-            else:
-                _logger.info("%s: setting desc for %s to %s", self.device.name, data.index, data.alias)
+            elif data.alias:
+                _logger.info("%s: setting desc for %r to %r", self.device.name, data.index, data.alias)
```

Comment:
This just tries to be consistent with the log statement preceding it.  It is likely not necessary to use `%r` for `data.index`, but it is useful to log a Python representation for `data.alias`, in case it is either empty or contains special characters.

Reply:
Nevertheless, for an integer like `data.index`, `%s` and `%r` should be identical.